frontend/src/index.js

```diff
@@ -1,15 +1,18 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import App from "./App";
-import reportWebVitals from "./reportWebVitals";
-import GlobalStyle from "./style/GlobalStyle";
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import GlobalStyle from './style/GlobalStyle';
+import { BrowserRouter as Router } from 'react-router-dom';
 
 ReactDOM.render(
   <React.StrictMode>
     <GlobalStyle />
-    <App />
+    <Router>
+      <App />
+    </Router>
   </React.StrictMode>,
-  document.getElementById("root")
+  document.getElementById('root')
```

Comment:
when making changes to your styling (e.g. using prettify) please consider making them in a separate pull request which does not need any reviews - this will keep your PRs shorter and easier to read